client/Client.js

```diff
@@ -2,12 +2,17 @@
  * @fileoverview Client.js handles client-side sockets and events.
  */
 
+const SERVER_WS_LOCATION = 'ws://3.144.98.109/nodejs/'; //not permanent, but where it's located now
+let socket = new WebSocket(SERVER_WS_LOCATION);
+socket.onopen = onOpen(socket);
```

Comment:
Does this code work? onOpen might not take the socket as a param. I never tested it, just wrote it on the design doc.